Samples/AppLifecycle/Activation/cpp-console-unpackaged/CppWinRtConsoleActivation/CppWinRtConsoleActivation.vcxproj

```diff
@@ -44,9 +45,6 @@
   </ItemGroup>
   <PropertyGroup Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
-    <PlatformToolset>v140</PlatformToolset>
```

Comment:
We should probably leave in the `PlatformToolset` property since the samples are available as individual zip downloads through the samples browser, for example: https://docs.microsoft.com/samples/microsoft/windowsappsdk-samples/applifecycle/

If folks download any of the samples as a zip file, the Build.Common.Cpp.props file won't be available

Reply:
I think in that case VS will alert you that you don't have the correct version installed and will ask you to do it. 
I feel strongly about not having it explicitly defined here, since having it currently breaks the builds done in the internal repo, and will continue to do so in future efforts of having CI builds or other pipeline runs. Do you have any advice on how to tackle this?

Reply:
I see! If VS alerts you then I would say we don't need to set the property then. Would be good to verify with one of the C++ samples in https://docs.microsoft.com/en-us/samples/browse/?expanded=windows&products=windows-app-sdk

Reply:
Good idea, I'll verify

Reply:
After trying, this is what VS shows right after opening the solution 
```C:\Users\er\Downloads\ResourceManagement\cpp-console-unpackaged\console_unpackaged_app.vcxproj : warning  : The build tools for Visual Studio 2010 (v100) cannot be found. To build using the Visual Studio 2019 (v142) build tools, either click the Project menu or right-click the solution, and then select "Retarget Solution". ```

Reply:
Is there any way to fallback on the VS 2022 version of the build tools? It seems confusing to have a message regarding Visusal Studio 2010?

Reply:
I agree. However, I think this is something that (at least my) visual studio sets. I don't have anything related to VS2010 installed and I still got that. For example:

![image](https://user-images.githubusercontent.com/26077674/149236372-1ae72ca4-1f3f-4610-ae19-376ce9262972.png)